include/mbedtls/ecdh.h

```diff
@@ -97,21 +98,22 @@ int mbedtls_ecdh_gen_public( mbedtls_ecp_group *grp, mbedtls_mpi *d, mbedtls_ecp
  *                  implemented during the ECDH key exchange. The first core
  *                  computation is performed by mbedtls_ecdh_gen_public().
  *
+ * \see             ecp.h
+ *
+ * \note            If \p f_rng is not NULL, it is used to implement
+ *                  countermeasures against potential elaborate timing
```

Comment:
Suggestion: Don't restrict to timing attacks, but instead say `... it is used to implement countermeasures against side-channel attacks. For more information, ...`. @yanesca Ping.

Reply:
done